sponsorship_compassion/models/res_partner.py

```diff
@@ -416,15 +416,58 @@ def _random_str():
                 "category_id": [(5, 0, 0)],
                 "comment": False,
                 "active": False,
+                "city": False,
+                "zip": False,
+                "commercial_company_name": False,
+                "company_name": False,
+                "contact_address": False,
+                "display_name": False,
+                "email_formatted": False,
+                "email_normalized": False,
+                "short_address": False,
+                "partner_latitude": False,
+                "partner_longitude": False,
+                "age": False,
```

Comment:
Age is a computed field, dependent on the birthdate field.
```suggestion
                "birthdate_date: False,
```